server/conf/application.test.conf

```diff
@@ -19,8 +19,13 @@ azure.blob.account = "my awesome azure account name"
 civiform_applicant_idp = "disabled"
 
 # address service area validation relative path for testing
-esri_find_address_candidates_url = ""
+esri_find_address_candidates_url = []
 esri_address_service_area_validation_urls = ["/query"]
+esri_address_service_area_validation_labels = ["Seattle"]
+esri_address_service_area_validation_ids = ["Seattle"]
+esri_address_service_area_validation_attributes = ["CITYNAME"]
```

Comment:
These are here because I found if I was testing with multiple options set in `application.dev.conf` the tests would fail because of a mismatch with the urls. Could have done it separately, but not going to bother.